engine/src/cmd/unit_csv.cpp

```diff
@@ -1088,162 +850,142 @@ void Unit::LoadRow( CSVRow &row, string modification, string *netxml )
+
+    graphicOptions.MinWarpMultiplier = UnitCSVFactory::GetVariable(unit_key, "Warp_Min_Multiplier", 1.0f);
+    graphicOptions.MaxWarpMultiplier = UnitCSVFactory::GetVariable(unit_key, "Warp_Max_Multiplier", 1.0f);
+
+    energy.Set( UnitCSVFactory::GetVariable(unit_key, "Primary_Capacitor", 0.0f));
+    recharge   = UnitCSVFactory::GetVariable(unit_key, "Reactor_Recharge", 0.0f);
+    jump.drive = UnitCSVFactory::GetVariable(unit_key, "Jump_Drive_Present", 0) ? -1 : -2;
+    jump.delay = UnitCSVFactory::GetVariable(unit_key, "Jump_Drive_Delay", 0);
+    forcejump = UnitCSVFactory::GetVariable(unit_key, "Wormhole", false);
+    graphicOptions.RecurseIntoSubUnitsOnCollision = UnitCSVFactory::GetVariable(unit_key, "Collide_Subunits", graphicOptions.RecurseIntoSubUnitsOnCollision ? true : false) ? 1: 0;
+    jump.energy = UnitCSVFactory::GetVariable(unit_key, "Outsystem_Jump_Cost", 0.0f);
+    jump.insysenergy = UnitCSVFactory::GetVariable(unit_key, "Warp_Usage_Cost", 0.0f);
     if (WCfuelhack) fuel = warpenergy = warpenergy+jump.energy*0.1f;       //this is required to make sure we don't trigger the "globally out of fuel" if we use all warp charges -- save some afterburner for later!!!
-    afterburnenergy  = ::stof( OPTIM_GET( row, table, Afterburner_Usage_Cost ), 32767 );
-    afterburntype    = ::stoi( OPTIM_GET( row, table, Afterburner_Type ) );     //type 1 == "use fuel", type 0 == "use reactor energy", type 2 ==(hopefully) "use jump fuel" 3: NO AFTERBURNER
-    limits.yaw = ::stof( OPTIM_GET( row, table, Maneuver_Yaw ) )*VS_PI/180.;
-    limits.pitch     = ::stof( OPTIM_GET( row, table, Maneuver_Pitch ) )*VS_PI/180.;
-    limits.roll      = ::stof( OPTIM_GET( row, table, Maneuver_Roll ) )*VS_PI/180.;
+    afterburnenergy  = UnitCSVFactory::GetVariable(unit_key, "Afterburner_Usage_Cost", 32767.0f);
+    afterburntype    = UnitCSVFactory::GetVariable(unit_key, "Afterburner_Type", 0); //type 1 == "use fuel", type 0 == "use reactor energy", type 2 ==(hopefully) "use jump fuel" 3: NO AFTERBURNER
+    limits.yaw = UnitCSVFactory::GetVariable(unit_key, "Maneuver_Yaw", 0.0f) *VS_PI/180.0;
+    limits.pitch     = UnitCSVFactory::GetVariable(unit_key, "Maneuver_Pitch", 0.0f) *VS_PI/180.0;
+    limits.roll      = UnitCSVFactory::GetVariable(unit_key, "Maneuver_Roll", 0.0f) *VS_PI/180.0;
+
+    YawPitchRollParser(unit_key, "Yaw_Governor", "Yaw_Governor", "Yaw_Governor", computer.max_yaw_right, computer.max_yaw_left);
+    YawPitchRollParser(unit_key, "Pitch_Governor", "Pitch_Governor_Up", "Pitch_Governor_Down", computer.max_pitch_up, computer.max_pitch_down);
+    YawPitchRollParser(unit_key, "Roll_Governor", "Roll_Governor_Right", "Roll_Governor_Left", computer.max_roll_right, computer.max_roll_left);
+
+    static float game_accel = GameConfig::GetVariable( "physics", "game_accel", 1.0 );
+    static float game_speed = GameConfig::GetVariable( "physics", "game_speed", 1.0 );
```

Comment:
did you mean `1.0f` on these since you're casting to a float?

Reply:
Fixed.